dashboard/app/controllers/user_level_interactions_controller.rb

```diff
@@ -48,7 +48,7 @@ def user_level_interaction_params
     user_level_interaction_params[:user_id] = current_user.id
     user_level_interaction_params[:school_year] = school_year
     unit = Unit.find(user_level_interaction_params[:script_id])
-    version_year = unit.version_year
+    version_year = unit.get_course_version.key
```

Comment:
This was an oversight from when I was trying out different ways to get version_year that should've gone in with https://github.com/code-dot-org/code-dot-org/pull/62288 😬 Worst case scenario is we weren't logging clicks on help&tips that we should've been from between the time #62288 hit prod and when this change goes in. I'm not worried about it though since we're just starting out.

Reply:
agree with not backfilling to fix the data. however this still needs to work for units not yet in courses, such as newly created units to be added to a course later:
```suggestion 
    version_year = unit.get_course_version&.key
```